src/com/lftechnology/java/training/dipak/shufflenumbers/ShuffleNumbers.java

```diff
@@ -0,0 +1,128 @@
+
+package com.lftechnology.java.training.dipak.shufflenumbers;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p> Creates an array list with Integer objects containing the numbers of the
+ * range provided by the user. It then shuffles the list and selects the values
+ * of certain range provided by user. Finally, it sorts the selected values and
+ * prints them out. </p>
+ * 
+ * @author Dipak Thapa <dipakthapa@lftechnology.com>
+ */
+
+public class ShuffleNumbers {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(ShuffleNumbers.class.getName());
+	private List<Integer> integerArray = new ArrayList<Integer>();
+	private List<Integer> sortedSubArray = new ArrayList<Integer>();
+
+	/**
+	 * <p>This function fills the arraylist with the numbers from the
+	 * lowerRange(inclusive) to upperRange(inclusive)</p>
+	 * 
+	 * @param sc
+	 * @param lowerRange
+	 * @param upperRange
+	 * @author Dipak Thapa <dipakthapa@lftechnology.com>
+	 */
+	public void getNumbers(Scanner sc, int lowerRange, int upperRange) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "sc". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)